galasa-ui/src/tests/components/TestRunsTabs.test.tsx

```diff
@@ -659,6 +659,44 @@ describe('TestRunsTabs Component', () => {
       });
     });
 
+    test('sorts by date in ascending order', async () => {
```

Comment:
Shouldn't the path to the test file roughly match the path to the implementation file ?
eg: galasa-ui/src/tests/components/test-runs/TestRunsTabs.test.tsx ?

If a folder for the source code helps organise the product code into natural logical pieces, the same segregation should be used for test code also ?

Reply:
Ok. Lets merge this one in and do that in a separate PR as you say.